src/BaseBallGame.js

```diff
@@ -1,49 +1,88 @@
+/* global MissionUtils */
 export default class BaseballGame {
-  /** @type {(number, number) => string} */
-  static play(computerInputNumbers, userInputNumbers) {
-    /** @type {number} */
-    let strikeCount = 0;
-    /** @type {number} */
-    let ballCount = 0;
-
-    for (let i = 0; i < 3; i += 1) {
-      /** @type {string} */
-      const result = this.compareNumberWithAnswers(computerInputNumbers, userInputNumbers[i], i);
-      if (result === 'strike') {
-        strikeCount += 1;
-      } else if (result === 'ball') {
-        ballCount += 1;
-      }
+  #answerNumbers = [];
+
+  #userNumbers = [];
+
+  #strikeCount = 0;
+
+  #ballCount = 0;
+
+  #result = '';
+
+  /** @type {(string) => bool} */
+  play(userInput) {
+    if (this.#convertInput(userInput) === false) {
```

Comment:
```suggestion
    if (!this.#convertInput(userInput)) {
```
falsy한 값을 판별할 때 지금 방식처럼 값을 명시하는 것도 좋아보이는데, 
js에서는 대부분 생략한답니다!

Reply:
저도 ! 사용을 선호하는 편인데 처음 배우는 언어이다 보니 보다 확실해 보이도록 === false 를 사용했었습니다. 다음부턴 !를 사용하도록 하겠습니다!